pkg/ee/cluster-backup/controller.go

```diff
@@ -323,6 +324,7 @@ func (r *reconciler) undeployClusterBackupUserClusterCRDs(ctx context.Context, u
 		LabelSelector: labels.SelectorFromSet(
 			map[string]string{
 				"component": "velero",
+				appskubermaticv1.ApplicationManagedByLabel: resources.ClusterBakcupControllerName,
```

Comment:
This should apply to the other resources as well, else we'll still delete Deployments and DaemonSets we never managed.